client/components/domains/trademark-claims-notice/index.jsx

```diff
@@ -0,0 +1,563 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { defer, get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import { composeAnalytics, recordGoogleEvent, recordTracksEvent } from 'state/analytics/actions';
+import HeaderCake from 'components/header-cake';
+import Button from 'components/button';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class TrademarkClaimsNotice extends React.Component {
+	static propTypes = {
+		isSignupStep: PropTypes.bool,
+		onAccept: PropTypes.func,
+		onReject: PropTypes.func,
+		recordAcknowledgeTrademarkButtonClickInTrademarkNotice: PropTypes.func,
```

Comment:
The `InTrademarkNotice` suffix seems unnecessary verbose - especially since we don't have any other buttons that can do the same, right?